docs/FAQ.md

```diff
@@ -42,22 +48,22 @@ this document. Few of answers assumes that the MCM is being used in conjuction w
 
 # Basics
 
-### What is Machine Controller Manager ?
-
-Machine controller manager aka MCM, is a bunch of controllers which is used to manage the lifecycle of the worker-machines. It reconciles a set of CRDs such as `Machine`, `MachineSet`, `MachineDeployment` which depicts the functionality of `Pod`, `Replicaset`, `Deployment` of the core kubernetes respectively. Read more about it at [README](https://github.com/gardener/machine-controller-manager/tree/master/docs).
+### What is Machine Controller Manager?
 
-* It is also used at gardener to manage the kubernetes-nodes of the shoot-cluster. By design it can be used independent of the gardener itself.
+Machine controller manager aka MCM, is a bunch of controllers which is used to manage the lifecycle of the worker-machines. It reconciles a set of CRDs such as `Machine`, `MachineSet`, `MachineDeployment` which depicts the functionality of `Pod`, `Replicaset`, `Deployment` of the core Kubernetes respectively. Read more about it at [README](https://github.com/gardener/machine-controller-manager/tree/master/docs).
 
-### Why is my machine deleted ?
+* It is also used at Gardener to manage the Kubernetes nodes of the shoot cluster. By design, it can be used independent of Gardener.
 
-A machine could be deleted by MCM, generally for 2 reasons.
+### What are different sub-controllers in MCM?
```

Comment:
It is usually the practice to keep the definitions or content related to a particular component to be same across documents. So, I copy pasted the same definition from Design/README here so as to maintain consistency.